### ISSUE TITLE
Winner cards are not keyboard accessible

### ISSUE THREAD

**Issue by alexgibson** [mozilla/rise25#19]
STR:

1. Open https://rise25.mozilla.org/
2. Tab to "See the Winners" and press enter
3. Tab to "Activists" (or any other category) and press enter.
4. Press tab to try and navigate to a winner.

Expected results:

- Upon pressing tab the winner should be focused. 
- Pressing enter again should display the winner's bio and shift focus to the bio. 
- Pressing enter again should close the bio.
- Pressing tab should shift focus to the next winner.
- And repeat

Actual results

Focus is shifted to the Mozilla logo in the footer, because no other page content is accessible via keyboard.
